packages/squiggle-lang/src/rescript/Reducer/Reducer_ErrorValue.res

```diff
@@ -50,7 +53,7 @@ let errorToString = err =>
   | RENotAFunction(valueString) => `${valueString} is not a function`
   | RERecordPropertyNotFound(msg, index) => `${msg}: ${index}`
   | RESymbolNotFound(symbolName) => `${symbolName} is not defined`
-  | RESyntaxError(desc) => `Syntax Error: ${desc}`
+  | RESyntaxError(desc, _) => `Syntax Error: ${desc}`
```

Comment:
Probably should've returned a location here but I wasn't even sure where `errorToString` is used and was too busy to check.

Reply:
I think it's worth it, if you can get around to it.